app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="form-container">
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="form-group mb-3">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ Required ] I think it would be nice if the styling for this `registration` form is similar to the styling provided on the project wireframe for the `login` form. Currently, your design varies from the wireframe in the following ways.
   
   - [ ] The `Sign up button` should have the same width as the `email` and `password` fields 
   - [ ] The `email`, `name`, and `password` input fields should have `placeholder` texts instead of the labels.